core/src/main/java/org/apache/accumulo/core/spi/balancer/HostRegexTableLoadBalancer.java

```diff
@@ -466,6 +475,13 @@ public long balance(BalanceParameters params) {
           LOG.trace("Sample up to 10 outstanding migrations: {}", limitTen(migrations));
         }
         return minBalanceTime;
+      } else if (!myConf.concurrentTables) {
+        LOG.warn("Not balancing tables due to {} existing migrations and {}} is set to false",
```

Comment:
Should this be at debug?  It seems that it will be common and would be expected to sort itself out eventually.  It would only be a concern is it was persistent?  Could the message include something to show that it may be making progress?  Current id being processed?  

Is this a log message that quickly flood the log?

Reply:
Yeah this should likely be debug or trace, I just copied it from above and didn't pay attention but that would definitely flood things pretty quickly and it is expected.

Reply:
This seems another case where we really could benefit from being able to log 1 of Number of 1 every T seconds occurrences of the same message.  It is not helpful to keep repeating that same message frequently, but having it appear occasionally helps troubleshoot cases where something in not making progress - might be out of scope for this, but could be worthwhile.